### ISSUE TITLE
Release as a Unity Package

### ISSUE THREAD

**Issue by thelucre** [VeryHotShark/First-Person-Controller-VeryHotShark#10]
This is a great asset. Thank you so much! Would it be possible to package up the required assets into a Unity Package for easy consumption?

**Comment by Endi24** [comment]
Second this. In order to import this to an existing project it just does not work.

**Comment by Glaas** [comment]
I agree with all these people ! It's manageable to import this to an existing project and make it work properly, but it does require some work

Also maybe enhance the documentation and how to use the controller ? If you do that, I wouldn't even mind paying for it :)
